.buildkite/pipeline.yml

```diff
@@ -36,8 +47,12 @@ steps:
             - "requirements/**"
             config:
               label: "🔨 MySQL"
+              env:
+                - PYTHON_VERSION=3.10
```

Comment:
I assume the plugin config doesn't automagically work with `matrix` and that's why this is all staying single version for now?

Reply:
And if it is gonna be one version, should it be 3.11, to align with the version we actually publish from our dockerfiles?

Reply:
Official plugin docs don't mention matrix support, unfortunately, so I skipped it here. Updating to 3.11 makes sense!